fieldservice/demo/fsm_location.xml

```diff
@@ -24,11 +24,10 @@
         <field name="email">tp@email.com</field>
         <field name="partner_id" eval="ref('test_loc_partner')"/>
         <field name="owner_id" eval="ref('test_loc_partner')"/>
-        <field name="customer_id" eval="ref('test_loc_partner')"/>
```

Comment:
Why is the customer being removed?

Reply:
@dreispt It is removed because When I applied format-patch and I found that some file code is not updated based on 12.0.2.4.0 fieldservice module code.

So I have replaced the latest code here (In this PR)

You can see the customer being removed in this commits https://github.com/OCA/field-service/commit/280039a676f259024fb6eba5b3863dc4ccca65af

Reply:
Yes, these were previously removed